config/migrations/2025/20250717144827-filter-form-concept-schemes/20250717144827-db-cleanup-add-organizations-to-concept-scheme.sparql

```diff
@@ -0,0 +1,34 @@
+PREFIX cleanup: <http://mu.semte.ch/vocabularies/ext/cleanup/>
+PREFIX mu:      <http://mu.semte.ch/vocabularies/core/>
+PREFIX dcterms: <http://purl.org/dc/terms/>
+
+INSERT DATA {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    <http://data.lblod.info/job/id/7dafe8eb-de89-49c4-9bac-bcaf0222fa3c> a cleanup:Job;
+      mu:uuid "7dafe8eb-de89-49c4-9bac-bcaf0222fa3c";
+      dcterms:title "Compensating missing rdf:Type on some formNodes in some subsidy forms";
+      cleanup:randomQuery """
+        INSERT {
+          GRAPH ?g{
+            ?organization <http://www.w3.org/2004/02/skos/core#inScheme> <http://data.lblod.info/id/conceptscheme/EconomischeActoren>
+          }
+        } WHERE {
+          GRAPH ?g {
```

Comment:
This will also update the publication graph `<http://mu.semte.ch/graphs/landing-zone/op-public>` we don't want that. That's the untouched source of truth reflecting the state from the master data op.
I think you can just update `http://mu.semte.ch/graphs/public`

Reply:
the initial migration in this PR seems right config/migrations/2025/20250717144827-filter-form-concept-schemes/20250717144826-add-concept-to-organizations.sparql